src/wp-includes/media.php

```diff
@@ -1748,6 +1760,50 @@ function wp_img_tag_add_loading_attr( $image, $context ) {
 	return $image;
 }
 
+/**
+ * Adds `width` and `height` attributes to an `img` HTML tag.
+ *
+ * @since 5.5.0
+ *
+ * @param string $image         The HTML `img` tag where the attribute should be added.
+ * @param string $context       Additional context to pass to the filters.
+ * @param int    $attachment_id Image attachment ID.
+ * @return string Converted 'img' element with 'width' and 'height' attributes added.
+ */
+function wp_img_tag_add_width_and_height_attr( $image, $context, $attachment_id ) {
+	$image_src         = preg_match( '/src="([^"]+)"/', $image, $match_src ) ? $match_src[1] : '';
```

Comment:
Wondering if it will need to capture the quote character here, perhaps? See https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/media.php#L1734. The only place I've seen single quotes for attributes (for now) is in literal img tag in js inside post_content.

In nearly all cases it will be a double quote, and it currently will skip adding the attributes when it is a single quote, so perhaps it's fine to have few "false negative" there.

Reply:
Also, perhaps the regex would be better to look for a space char before the `src`? Maybe ` '/\ssrc="([^"]+)"/'`?

Reply:
Thinking more about this, instead of capturing the quote here, maybe better to remove that code from `wp_img_tag_add_loading_attr()`. It's best to "sync" how the two functions work as the attributes are closely related.

Reply:
Just pushed https://github.com/WordPress/wordpress-develop/pull/358/commits/78a3b5bdba6d0d51d8218a747ff2c2178fa09a08 to remove the single quote logic, I agree that was probably overkill for there, since WordPress doesn't use single quotes there and the other related functions don't cater for that either.